plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -107,6 +107,10 @@ func init() {
 	factory.RegisterPriorityFunction2("ImageLocalityPriority", priorities.ImageLocalityPriorityMap, nil, 1)
 	// Optional, cluster-autoscaler friendly priority function - give used nodes higher priority.
 	factory.RegisterPriorityFunction2("MostRequestedPriority", priorities.MostRequestedPriorityMap, nil, 1)
+	// Prioritizes nodes that satisfy pod's resource limits
+	if !utilfeature.DefaultFeatureGate.Enabled(features.ResourceLimitsPriorityFunction) {
```

Comment:
why is this !utilfeature... instead of utilfeature... ?

Reply:
I will fix it, seems i forgot to revert it after testing. thanks for catching this.